packages/ui/components/form-core/test-suites/ValidateMixinFeedbackPart.suite.js

```diff
@@ -622,6 +626,49 @@ export function runValidateMixinFeedbackPart() {
       });
     });
 
+    it('".getMessage()" gets .inputMode defined on Validator config', async () => {
+      const constructorValidator = new MinLength(4, {
+        // eslint-disable-next-line no-promise-executor-return
+        inputMode: 'select',
+      });
+      const ctorValidator = /** @type {typeof MinLength} */ (constructorValidator.constructor);
+      const spy = sinon.spy(ctorValidator, 'getMessage');
+
+      const el = /** @type {ValidateElement} */ (
+        await fixture(html`
+        <${tag}
+          .submitted=${true}
+          .validators=${[constructorValidator]}
+          .modelValue=${'cat'}
+          .fieldName=${new Promise(
+            (
+              resolve, // eslint-disable-next-line no-promise-executor-return
+            ) => resolve('myField'),
+          )}
+        >${lightDom}</${tag}>
+      `)
+      );
+      await el.updateComplete;
+      await el.feedbackComplete;
+
+      // ignore fieldName Promise as it will always be unique
+      const compare = spy.args[0][0];
+      delete compare?.config?.fieldName;
+      expect(compare).to.eql({
+        config: {
+          inputMode: 'select',
```

Comment:
Not really sure if developers should be able to configure this.
It's up to the developer of a FormControl to make it accessible. 

In case a developer wants to change the message, he should provide the getMessage method